ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -48,6 +63,37 @@
     }
   }
 
+  &__more-options-beta {
+    line-height: $typography-sm-line-height;
+
+    a {
+      color: $color-dark-blue;
+      text-decoration: none;
+      border: 0;
+      font-size: 1.1rem;
```

Comment:
Sorry just noticed this one - we have a variable for this now

Reply:
@jlhdxw thanks for spotting updated